react/StyleGuideProvider/StyleGuideProvider.js

```diff
@@ -7,6 +7,10 @@ import classnames from 'classnames';
 import Helmet from 'react-helmet';
 import ScreenReaderOnly from '../ScreenReaderOnly/ScreenReaderOnly';
 
+if (typeof window !== 'undefined') {
+  require('smoothscroll-polyfill').polyfill();
```

Comment:
I might be a bit biased about this because profile has it's own scroll library as it's a huge part of our experience, which won't match this. We actually wrote it ourselves as we had specific requirements that this simple polyfill doesn't achieve. However, I can see your point.